stock_quant_package_fast_move/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Fast Package Move Between Locations",
+    "version": "16.0.1.0.0",
+    "category": "Inventory/Inventory",
+    "summary": "Move packages between locations directly from the 'Package' menu",
+    "depends": ["stock_picking_move_package_to_package"],
```

Comment:
Hi @angelinaanaki why do we need to depend on this module?
Are we using any of its functions?